is-greater-than-one/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+        function isGreaterThanOne(fraction){
+            if(eval(fraction) >= 1)
```

Comment:
Hey Brandon, while this works, I need to bring up the evils of eval lol I will address the class about it as a group and thank you for reminding me of it. In short, eval is dangerous because it allows people to run malicious scripts in your code. It should not ever be used. It is a very useful tool, but bad people have ruined it for the rest of us! Smart solution though!